Missionframework/modules/23_logistic/fnc/fn_logi_refreshTargets.sqf

```diff
@@ -0,0 +1,61 @@
+/*
+    KPLIB_fnc_logi_refreshTargets
+
+    File: fn_logi_refreshTargets.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2019-01-25
+    Last Update: 2019-02-21
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Refreshes the vehicle list for the given dialog.
+
+    Parameter(s):
+        _dialogId   - The dialog id     [Number, defaults to 0]
+        _ctrlId     - The control id    [Number, defaults to 0]
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+params [
+    ["_dialogId", 0, [0]],
+    ["_ctrlId", 0, [0]]
+];
+
+// Dialog controls
+private _dialog = findDisplay _dialogId;
+private _ctrlVehicleList = _dialog displayCtrl _ctrlId;
+
+lbClear _ctrlVehicleList;
+
+// Get all FOB vehicles
+private _nearFOB = KPLIB_sectors_fobs select {(player distance2D (getMarkerPos _x)) < KPLIB_param_fobRange};
```

Comment:
There is an FOB variable set on player if he enters one, no need to search for it manually.
```suggestion
private _nearFOB = player getVariable ["KPLIB_fob", ""];
```
Maybe it could be moved to common function so it becomes:
```sqf
private _nearFOB = player call KPLIB_fnc_common_getPlayerFob;
```